ferris_elf/__init__.py

```diff
@@ -314,6 +314,43 @@ async def formatted_scores_for(
 
     return builder.getvalue()
 
+async def formatted_best(
+    author: Union[discord.User, discord.Member],
+    bot: discord.Client,
+    db: Database,
+    part: int,
+) -> str:
+    builder = io.StringIO()
+
+    # If the message was not sent in a DM, get the author's guild.
+    if isinstance(author, discord.Member):
+        # FIXME [NP]: Is this redundant because we have the `author.guild` already?
+        guild = bot.get_guild(author.guild.id)
+    else:
+        guild = None
+
+    for opt_day, _opt_part, opt_user, opt_bench_time in db.get_best_lb(part):
+        if opt_day is None or _opt_part is None or opt_user is None or opt_bench_time is None:
+            continue
+
+        user = int(opt_user)
+
+        # if the aoc command was sent in a guild that isnt the guild of the user we have here, then using <@id>
+        # will render as <@id>, instead of as @person, so we have to fallback to using the name directly
+        if guild is None or guild.get_member(user) is None:
+            userobj = bot.get_user(user) or await bot.fetch_user(user)
+            if userobj:
+                builder.write(
+                    f"\td{opt_day}: {escape_markdown(userobj.name)} - **{ns(opt_bench_time)}**\n"
+                )
+            continue
+        builder.write(f"\td{opt_day}: <@{user}> - **{ns(opt_bench_time)}**\n")
```

Comment:
@bend-n as you are the formatting expert, any ideas on how to prettify the output?

<img width="446" alt="image" src="https://github.com/user-attachments/assets/869dfa56-5b41-4bc7-944b-874abe8451a1" />

Reply:
lol… i was just looking at this and cringing at the `-` while waiting for the notification to load.
do you think we could have it more like this?
![image](https://github.com/user-attachments/assets/1ba38499-3591-4190-b422-e8dab6cd1f4c)
could also keep it split, and just merge in the duplicates, such as (keeping it more like `aoc`'s result)
![image](https://github.com/user-attachments/assets/914e72ae-448e-4a93-a8a4-ffc4f254950e)
possibly removing the duplication of the day
![image](https://github.com/user-attachments/assets/2cfe9123-f849-49b5-874a-43483a8d2836)